nixos/modules/services/misc/jellyfin.nix

```diff
@@ -64,14 +64,16 @@ in
         AmbientCapabilities = "";
         CapabilityBoundingSet = "";
 
-        # ProtectClock= adds DeviceAllow=char-rtc r
-        DeviceAllow = "";
+        # allow graphic cards
+        DeviceAllow = [
+          "/dev/dri/*"
+        ];
+        SupplementaryGroups = [ "video" ];
+        PrivateDevices = false;
```

Comment:
Is there a reason to explicitly set it to false here? I'm not aware of an option set here that sets `PrivateDevices = true`.

Reply:
The main reason was that when keeping it set to true still does not work for me ... i have not investigated further

Reply:
as a note we should probably keep the comment original

`Disabled to allow Jellyfin to access hw accel devices endpoints`

Reply:
I think my wording was a bit ambiguous: I don't mind having PrivateDevices set to false, but since it should be the default behavior, why not remove this line entirely?